Update docs for version 0.9.9 (#278)
AequilibraE/qaequilibrae: docs/source/menus_in_detail/pathscomputation.rst

------------------

Having verified that the network seems to be in order, one can proceed to
perform traffic assignment, since we have a demand matrix.

.. image:: ../images/traffic_assignment.png
    :width: 350
    :align: center
    :alt: Calling assignment

The Traffic Assignment procedure tab looks like this!

.. image:: ../images/project_overview.png
    :width: 877
    :align: center
    :alt: Project overview

In the *Traffic Classes* tab you will create the traffic classes used in the project.

> First, select one of the available matrices (in \*.AEM format), and the matrix core

Reviewer: OMX are not allowed? I am pretty sure they are...

Reviewer: Do you have OMX installed?? That is supposed to show ONLY if openmatrix is not installed...

Author: In QGIS? I don't think so.. (so that's probably this). I'll fix the documentation

> that will be used for computation. For Sioux Falls example, we don't want to block

Reviewer: ```suggestion
that will be used for computation. For the Sioux Falls example, we don't want to block
```

> flow through centroids, but this might not always be the case. When you finish, just

Reviewer: ```suggestion
flow through centroids, but this is only necessary because regular nodes of the network are centroids. When you finish, just
```

    :align: center
    :alt: Setup assignment

> The result of the traffic allocation we just performed is stored in the results.sqlite

Reviewer: ```suggestion
The result of the traffic assignment we just performed is stored in the results.sqlite
```
